docs/source/migration.mdx

```diff
@@ -1611,6 +1618,7 @@ new ApolloServer({
 ## Renamed packages
 
 The following packages have been renamed in Apollo Server 4:
+ * `apollo-datasource-rest` is now [`@apollo/datasource-rest`](https://www.npmjs.com/package/@apollo/datasource-rest). The `@apollo/datasource-rest` package is compatible with both Apollo Server 3 and 4.
```

Comment:
you could be even stronger and somehow point out that the new version has no explicit dependency on apollo-server and can be used independently of it if you want? maybe not.

Reply:
I like the point about removing the explicit dependency!  How about:

> - `apollo-datasource-rest` is now [`@apollo/datasource-rest`](https://www.npmjs.com/package/@apollo/datasource-rest). The `@apollo/datasource-rest` package removes an explicit dependency on `@apollo/server` and is compatible with both Apollo Server 3 and 4.

Reply:
I guess the trick here though is that when we say "compatible with Apollo Server 3" we mean "you can use the same create-it-yourself-and-put-on-context pattern in AS3 that we document for AS4", not that you can slot it in exactly where you used the old one in AS3 (in `dataSources`).

Reply:
Oh, dang, that makes so much sense....maybe I'll just remove the "compatible with Apollo Server 3" part? Unless you feel like it's valuable to keep it in? If so, I can add that background in

Reply:
Yeah that's probably fine.

It's compatible with Apollo Server 3 in the sense that every package on npmjs.com that is unrelated to Apollo Server is compatible with Apollo Server 3.  `apollo-datasource-rest` is a package that is very tangentially related to Apollo Server 3; `@apollo/datasource-rest` is a package that is unrelated to Apollo server 3.